.github/workflows/add-to-board.yml

```diff
@@ -0,0 +1,47 @@
+name: Triage & route issues
+
+on:
+  issues:
+    types: [labeled]
+
+permissions:
+  issues: write
+  contents: read
+  projects: write
```

Comment:
unfortunately, `projects` doesn't exist. See https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token

We will need a PAT with org/projects write permissions.